diagnostic.md

```diff
@@ -7,40 +7,40 @@ What command do we use to do that? Use the **correct** URL from your fork of
 this repository in your answer.
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+<!-- git clone git@github.com:lisamsmith100/git-diagnostic.git -->
 ```
 
 2.What do you do after cloning a repository, before you start making any
 changes/additions?
 
-<!-- Remove this comment and place your answer here. -->
+<!-- create a branch on which I can do my work -->
 
 3.What command do we use to create a new branch? Name this branch `response`
     in your answer. Then, how do we switch to that branch?
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+<!-- git checkout -b response OR git branch response then git checkout response -->
 ```
 
 4.We just wrote some code. What command do we use to see a summary of the
     changes in our working directory?
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+<!-- git diff -->
 ```
 
 5.We want to prepare a change for a commit by adding a file to the staging
     area. What command do we use to stage a file named `diagnostic.md`?
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+<!-- git add diagnostic.md -->
 ```
 
 6.Once `diagnostic.md` is staged, we have to make a commit by `git commit`.
 What are the two formatting items you **need** to make up your commit message?
 
-<!-- Remove this comment and place your answer here. -->
+<!-- first letter in message = capitalized; first word = verb (imperative)   -->
```

Comment:
Really close here too Lisa. The two formatting items we need are a short title that summarizes what we've changed, and then a longer message beneath the title that describes in more detail what we've changed.

As far as your answer though you were right in that is the way we want our commit title's to be structured. Good job.